specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -0,0 +1,1290 @@
+        "Standard_F32s_v2",
+        "Standard_F4",
+        "Standard_F4s",
+        "Standard_F4s_v2",
+        "Standard_F64s_v2",
+        "Standard_F72s_v2",
+        "Standard_F8",
+        "Standard_F8s",
+        "Standard_F8s_v2",
+        "Standard_G1",
+        "Standard_G2",
+        "Standard_G3",
+        "Standard_G4",
+        "Standard_G5",
+        "Standard_GS1",
+        "Standard_GS2",
+        "Standard_GS3",
+        "Standard_GS4",
+        "Standard_GS4-4",
+        "Standard_GS4-8",
+        "Standard_GS5",
+        "Standard_GS5-16",
+        "Standard_GS5-8",
+        "Standard_H16",
+        "Standard_H16m",
+        "Standard_H16mr",
+        "Standard_H16r",
+        "Standard_H8",
+        "Standard_H8m",
+        "Standard_L16s",
+        "Standard_L32s",
+        "Standard_L4s",
+        "Standard_L8s",
+        "Standard_M128-32ms",
+        "Standard_M128-64ms",
+        "Standard_M128ms",
+        "Standard_M128s",
+        "Standard_M64-16ms",
+        "Standard_M64-32ms",
+        "Standard_M64ms",
+        "Standard_M64s",
+        "Standard_NC12",
+        "Standard_NC12s_v2",
+        "Standard_NC12s_v3",
+        "Standard_NC24",
+        "Standard_NC24r",
+        "Standard_NC24rs_v2",
+        "Standard_NC24rs_v3",
+        "Standard_NC24s_v2",
+        "Standard_NC24s_v3",
+        "Standard_NC6",
+        "Standard_NC6s_v2",
+        "Standard_NC6s_v3",
+        "Standard_ND12s",
+        "Standard_ND24rs",
+        "Standard_ND24s",
+        "Standard_ND6s",
+        "Standard_NV12",
+        "Standard_NV24",
+        "Standard_NV6"
+      ]
+    },
+    "KeyVaultSecretRef": {
+      "properties": {
+        "vaultID": {
+          "type": "string",
+          "description": "Key vault identifier."
+        },
+        "secretName": {
+          "type": "string",
+          "description": "The secret name."
+        },
+        "version": {
+          "type": "string",
+          "description": "The secret version."
+        }
+      },
+      "description": "Reference to a secret stored in Azure Key Vault.",
+      "required": [
+        "vaultID",
+        "secretName"
+      ]
+    },
+    "ContainerServiceServicePrincipalProfile": {
+      "properties": {
+        "clientId": {
+          "type": "string",
+          "description": "The ID for the service principal."
+        },
+        "secret": {
+          "type": "string",
+          "description": "The secret password associated with the service principal in plain text."
+        },
+        "keyVaultSecretRef": {
+          "$ref": "#/definitions/KeyVaultSecretRef",
+          "description": "Reference to a secret stored in Azure Key Vault."
+        }
+      },
+      "description": "Information about a service principal identity for the cluster to use for manipulating Azure APIs. Either secret or keyVaultSecretRef must be specified.",
+      "required": [
+        "clientId"
+      ]
+    },
+    "ContainerServiceMasterProfile": {
```

Comment:
Also not sure we need all or maybe some of the master profiles in AKS. If it has been this way in previous version, it is probably ok to not cleaning it for now, maybe create a task to document this cleaning work for future?

There might be some other similar items that is in the ACS model but not in the AKS model.

Reply:
It's been this way since AKS began, so I'd rather leave it for now. I'll make a task to consider cleaning it up in the future.